### PR TITLE
try deployment using cobbler if indicated

### PR DESCRIPTION
**Pull request by sjamgade**
so that the deployment route of cobbler can be tested

It uses the variable `usecobbler` to decide on the detour.

### CODE REVIEW

File: scripts/jenkins/ardana/ansible/init.yml [hunk 1]
```diff
@@ -131,6 +132,25 @@
...
+
+    when: usecobbler is defined and usecobbler|bool == True
+    args:
+      chdir: /var/lib/ardana/openstack/ardana/ansible
+    become: true
+    become_user: ardana
+
+  - name: Rebuild nodes for cobbler deployement
+    shell: |
+        ansible-playbook -vvv -i hosts/localhost bm-reimage.yml  ardanauser_password="linux"
```
Comment (aplanas) [inline scripts/jenkins/ardana/ansible/init.yml]: "ardanauser_password=linux" like in line 137?

**Comment by sjamgade** [comment]
the input_model is hosted here  `http://10.84.144.252/susefication/input_model/` and I plan to use ```wget --no-clobber --no-parent --recursive -A '*.yml' -nH --cut-dirs=2 http://10.84.144.252/susefication/input_model```

However the heat template needs to change for 5 nodes

File: scripts/jenkins/ardana/ansible/init.yml (hunk 1 above)
Comment (nicolasbock) [reply scripts/jenkins/ardana/ansible/init.yml]: Don't you need `-e ardanauser....` to pass that value into the environment?

**Comment by dirkmueller** [comment]
does it work? is it still WIP? can it be rebased?

**Comment by sjamgade** [comment]
oops, forgot about this one, I could not get the correct heat template, so closing this

**Pull request closed without merging**